packages/@react-spectrum/icon/package.json

```diff
@@ -33,7 +33,13 @@
     "@react-types/shared": "^3.0.0-rc.2"
   },
   "devDependencies": {
-    "@adobe/spectrum-css-temp": "^3.0.0-alpha.1"
+    "@adobe/spectrum-css-temp": "^3.0.0-alpha.1",
+    "@adobe/spectrum-css-workflow-icons": "^1.0.0",
```

Comment:
hmm.... we don't really have a good way to track this in relation to the icons we actually build, can we check the version that was used to build `@spectrum-icons/workflow`?

Reply:
@snowystinger is the version that was used to build spectrum-icons/workflow accessible anywhere?

Reply:
yarn.lock?
```
"@adobe/react-spectrum-workflow@^1.0.0":
  version "1.0.1"
```
though I guess yarn will attempt to make it so there is only one copy, so I think I was just being semi-silly with my comment (not that i really remember why I made it). we should put in some sort of lint rule at some point to make sure we don't end up with multiple versions of this though...